### DISCUSSION TITLE
Star catalogs, again

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2972] [Ideas]
According to the manual, Stellarium uses Hipparcos and Tycho 2 catalogs, as well as NOMAD, which combines Hipparcos, Tycho-2, UCAC2, USNO-B1.0 and 2MASS. Altogether, around 177 million registers. Probably all those but Hipparcos are now superseded by GAIA DR3, widely available. Sooner or later, Stellarium will provide ways to add or manage this and future ICRS catalogs.

Maybe it would be good to start by providing information on the source catalog in the side panel, so the user can assess how reliable is it.

**Comment by gzotti** [comment]
The information is available in the User Guide, and for now there is no option to switch. Why cluttering the screen?

**Reply by diegorodriguezperegrin** [reply]
Yes, there is currently no option to change or add catalogs. That's why Stellarium, as information-rich and visually impressive as it is, lags behind other planetarium programs in this regard.
